exercise1/exercise7/main.cpp

```diff
@@ -0,0 +1,26 @@
+/*Exercise 7 - Operator Precedence
+Write a program that will ask the user for three numbers (a,b,c) and output the following evaluations:
+a + 1 * b + 2 - c
+(a + 1) * (b + 2) - c
+*/
+#include <iostream>
+using namespace std;
+
+int main() 
+{
+	int a, b, c, x1, x2;
```

Comment:
There is no gain in declaring your variables before the rest of your code. Try to have them declared as close as possible or in the same line as the definition.